paywall/src/__tests__/data-iframe/cacheHandler.test.js

```diff
@@ -199,6 +231,131 @@ describe('cacheHandler', () => {
     })
   })
 
```

Comment:
these are the new test concepts: making sure behavior is correct when account changes or network changes